packages/zowe-explorer/__tests__/__unit__/command/mvsCommandHandler.unit.test.ts

```diff
@@ -12,16 +12,14 @@
 jest.mock("Session");
 
 import * as vscode from "vscode";
```

Comment:
There may be schemarios where the order of the `jest.mock` matters 😋 
I'm curious if we could move this one below the imports (since we did it for other test files in this PR) 😋 

I know this is just me being picky, 😓